core/health/rest/src/main/resources/openapi/health-check-api.yaml

```diff
@@ -0,0 +1,50 @@
+openapi: 3.0.1
+info:
+  version: 1.0.0
+  title: OpenNMS Health Check RESTful API
+  description: OpenNMS Health Check RESTful API
+  contact:
+    name: OpenNMS
+    url: 'http://www.opennms.com/'
+  license:
+    name: OpenNMS(R) Licensing
+    url: 'http://www.gnu.org/licenses/'
```

Comment:
This is pretty generic, it should probably point to https://github.com/OpenNMS/opennms/blob/develop/LICENSE.md instead

Reply:
Thanks, updated, please check. I also changed the branch that is going to merge from develop to release-28.x